app_dart/lib/src/service/build_status_provider.dart

```diff
@@ -56,14 +56,15 @@ class BuildStatusService {
       branch: branch,
     ).toList();
     if (statuses.isEmpty) {
-      return BuildStatus.failed;
+      return BuildStatus.success();
```

Comment:
Should this still be a failure? If Datastore is failing to return the tasks, this is an infra issue that should close the tree.

Reply:
Thanks, that was just an typo.  Fixed.